index.html

```diff
@@ -1790,19 +1793,19 @@
           When an <a>NFC device</a> comes in communication range,
           <ul>
             <li>
-              If <var>options.target</var> has the value <code>"tag"</code>,
+              If <var>target</var> has the value <code>"tag"</code>,
```

Comment:
If the <var>options</var> object's <var>target</var> member has the value <code>"tag"</code>,